adapter/pkg/config/log_types.go

```diff
@@ -57,15 +59,41 @@ func getDefaultLogConfig() *LogConfig {
 		AccessLogs: &accessLog{
 			Enable:  false,
 			LogFile: "/dev/stdout",
+			LogType: "text",
 			// Following default value of "ReservedLogFormat" is document in log_config.toml for references.
 			// Update log_config.toml if any changes are done here.
-			ReservedLogFormat: "[%START_TIME%]' '%ROUTE_NAME%' '%DYNAMIC_METADATA(envoy.filters.http.ext_authz:originalHost)%' " +
+			ReservedLogFormat: "[%START_TIME%]' '%DYNAMIC_METADATA(envoy.filters.http.ext_authz:originalHost)%' " +
```

Comment:
why have we removed the route Name here.

Reply:
If we add a new field in the middle of the access log, the fields right to it get shifted. So there would be two positions for old and new logs. If we are adding, we have to append it to the end.